CrewChiefV4/ui_text.txt

```diff
@@ -1058,12 +1058,8 @@ enable_iracing_pit_stop_commands_help = If you do not use iRacing pit stop comma
 enable_iracing_pit_stop_commands_filter = IRACING
 enable_iracing_pit_stop_commands_category = AUDIO_VOICE_AND_CONTROLLERS;PIT_STOPS_AND_MULTICLASS
 
-enable_tight_fuel_calc = Enable tight fuel calculations
-enable_tight_fuel_calc_help = Enable this if you don't want the app to add additional fuel when asking for how much fuel is needed to end the race
-enable_tight_fuel_calc_category = FUEL_TEMPS_AND_DAMAGES
-
-add_additional_fuel = Additional fuel to add to finish the race
-add_additional_fuel_help = The amount of extra fuel (liters or gallons) you want the app to add when "Use tight fuel calculations" is enabled
+add_additional_fuel = Additional fuel to add to finish the race(reserve)
+add_additional_fuel_help = The amount of laps of fuel you want the app to add as a reserve, set this value if you want to overwrite the buildin reserve. This value can be factions of a lap eg. 0.7
```

Comment:
Amount of extra fuel to add (in laps or fractions of a lap) when working out how much is needed to finish a race.